Userland/Libraries/LibCore/Stream.cpp

```diff
@@ -149,7 +149,7 @@ ErrorOr<void> File::open_path(StringView const& filename, mode_t permissions)
 #endif
 
     if (fd < 0) {
-        return Error::from_errno(errno);
+        return Error::from_syscall("open", -errno);
     }
```

Comment:
Hmm, can we simplify this whole diff with `Core::System` wrappers? :)

Reply:
Should I add the ones which are missing?